features/support/hooks.rb

```diff
@@ -0,0 +1,11 @@
+Before('@remove-generator-files') do
+  command = <<-COMMAND
+  rm -f \
+    tmp/example_app/lib/generators/my_generator \
+    tmp/example_app/lib/generators/my_generator/my_generator_generator.rb \
+    tmp/example_app/lib/generators/my_generator/USAGE \
+    tmp/example_app/lib/generators/my_generator/templates \
+    tmp/example_app/spec/generator/my_generator_spec.rb
+  COMMAND
+  system(command)
+end
```

Comment:
Is this necessary? Does cucumber not isolate the examples?

Reply:
Mmmh it seems yes. I had one issue with the command trying to create files be there already there. I just pushed changes. We will see.

Reply:
Not needed! Yes!!!